rust_snuba/src/processors/snapshots/rust_snuba__processors__tests__schemas@ingest-replay-events-ReplaysProcessor-ingest-replay-events__1__archive.json.snap

```diff
@@ -1,6 +1,6 @@
 ---
 source: src/processors/mod.rs
-description: "{\n  \"type\": \"replay_event\",\n  \"start_time\": 1702848658.558295,\n  \"replay_id\": \"834f314caae54030a1b0dc52b202f24a\",\n  \"project_id\": 1,\n  \"retention_days\": 90,\n  \"payload\": [\n    123, 34, 116, 121, 112, 101, 34, 58, 32, 34, 114, 101, 112, 108, 97, 121,\n    95, 101, 118, 101, 110, 116, 34, 44, 32, 34, 114, 101, 112, 108, 97, 121,\n    95, 105, 100, 34, 58, 32, 34, 56, 51, 52, 102, 51, 49, 52, 99, 97, 97, 101,\n    53, 52, 48, 51, 48, 97, 49, 98, 48, 100, 99, 53, 50, 98, 50, 48, 50, 102,\n    50, 52, 97, 34, 44, 32, 34, 116, 105, 109, 101, 115, 116, 97, 109, 112, 34,\n    58, 32, 49, 55, 48, 50, 56, 52, 56, 54, 53, 56, 46, 53, 53, 56, 50, 57, 53,\n    44, 32, 34, 105, 115, 95, 97, 114, 99, 104, 105, 118, 101, 100, 34, 58, 32,\n    116, 114, 117, 101, 125\n  ]\n}\n"
+description: "{\n  \"type\": \"replay_event\",\n  \"start_time\": 1702848658.558295,\n  \"replay_id\": \"834f314caae54030a1b0dc52b202f24a\",\n  \"project_id\": 1,\n  \"retention_days\": 90,\n  \"payload\": [\n    123, 34, 116, 121, 112, 101, 34, 58, 32, 34, 114, 101, 112, 108, 97, 121,\n    95, 101, 118, 101, 110, 116, 34, 44, 32, 34, 114, 101, 112, 108, 97, 121,\n    95, 105, 100, 34, 58, 32, 34, 56, 51, 52, 102, 51, 49, 52, 99, 97, 97, 101,\n    53, 52, 48, 51, 48, 97, 49, 98, 48, 100, 99, 53, 50, 98, 50, 48, 50, 102,\n    50, 52, 97, 34, 44, 32, 34, 116, 105, 109, 101, 115, 116, 97, 109, 112, 34,\n    58, 32, 49, 55, 48, 50, 56, 52, 56, 54, 53, 56, 46, 53, 53, 56, 50, 57, 53,\n    44, 32, 34, 105, 115, 95, 97, 114, 99, 104, 105, 118, 101, 100, 34, 58, 32,\n    116, 114, 117, 101, 44, 32, 34, 101, 118, 101, 110, 116, 95, 104, 97, 115,\n    104, 34, 58, 32, 34, 54, 54, 99, 98, 102, 49, 98, 97, 52, 53, 100, 98, 52,\n    97, 51, 48, 98, 52, 101, 102, 49, 50, 98, 98, 51, 48, 54, 101, 101, 99, 99,\n    49, 34, 125\n  ]\n}\n"
```

Comment:
how come the contents of some of them changed, not just the name?

Reply:
i think this is plainly a different example. git thinks this is a rename, but it's actually just not